packages/shared/lib/core/profile-manager/actions/updateStronghold.ts

```diff
@@ -0,0 +1,33 @@
+import { get } from 'svelte/store'
+
+import { getStorageDirectoryOfProfile } from '@core/profile'
+import { activeProfileId, updateActiveProfile } from '@core/profile/stores'
+import { destroyProfileManager } from '@core/profile-manager/actions'
+import { api } from '@core/profile-manager/api'
+import { getSecretManagerPath } from '@core/profile-manager/utils'
```

Comment:
These should be relative import paths; we don't want to use the path alias if we're in the same module.

Reply:
Oh sorry, I've understood it wrongly, fixing it right now